esptool overlap error

Hi
i'm using sming with docker and just compiling HttpServer_WebSockets, but esptool throws error.
I have not changed anything in example.

`/opt/esp-open-sdk/esptool/esptool.py -p /dev/ttyUSB0 -b 115200 write_flash -ff 40m -fm qio -fs 4m 0x00000 out/firmware/0x00000.bin 0x0a000 out/firmware/0x0a000.bin  0xB000 out/firmware/spiff_rom.bin`
`WARNING: Flash size arguments in megabits like '4m' are deprecated.
Please use the equivalent size '512KB'.`
`Megabit arguments may be removed in a future release.`
`usage: esptool write_flash [-h] [--flash_freq {40m,26m,20m,80m}]`
                           `[--flash_mode {qio,qout,dio,dout}]`
                          ` [--flash_size FLASH_SIZE] [--ucIsHspi]`
                           `[--ucIsLegacy] [--no-progress] [--verify]`
                           `[--compress | --no-compress]`
                           `<address> <filename> [<address> <filename> ...]`
`esptool write_flash: error: argument <address> <filename>: Detected overlap at address: 0xb000 for file: out/firmware/spiff_rom.bin`
`/workspace/Sming/Sming/Makefile-project.mk:449: recipe for target 'flash' failed
make: *** [flash] Error 2`

--- a/Sming/Makefile-linux.mk
+++ b/Sming/Makefile-linux.mk
@@ -11,6 +11,6 @@ SDK_TOOLS	 ?= $(SDK_BASE)/tools
 # Other tools mappings
 ESPTOOL		 ?= $(ESP_HOME)/esptool/esptool.py
 KILL_TERM    ?= pkill -9 -f "$(COM_PORT) $(COM_SPEED_SERIAL)" || exit 0
-GET_FILESIZE ?= stat --printf="%s"
+GET_FILESIZE ?= stat -c %s
 TERMINAL     ?= python -m serial.tools.miniterm $(COM_PORT) $(COM_SPEED_SERIAL) $(COM_OPTS)
 MEMANALYZER  ?= python $(SMING_HOME)/../tools/memanalyzer.py $(OBJDUMP)

